### PR TITLE
update README to clarify install steps

### PR DESCRIPTION
**Pull request by taravancil**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,16 +4,30 @@ A Web server for [Dat](https://datprotocol.com) and HTTPS.
...
 
  - Serve sites over Dat at `dat://{subdomain}.{yourdomain.com}`.
  - Rehost those sites over `https://{subdomain}.{yourdomain.com}`.
- - Get TLS certs automatically with Lets Encrypt.
+ - Get TLS certs automatically with Let's Encrypt.
  - (Optionally) Auto-redirect from https -> dat.
 
+## Getting started
+
+### Upload your website to a Dat archive
```
Comment (pfrazee) [inline README.md]: Is 'upload' the right word? Host, serve?

File: README.md [hunk 2]
```diff
@@ -4,16 +4,30 @@ A Web server for [Dat](https://datprotocol.com) and HTTPS.
...
+
+You can use the [Dat CLI](https://www.npmjs.com/package/dat) or the [Beaker
+Browser](https://github.com/beakerbrowser/beaker).
+
+After uploading your site to a Dat archive, identify the archive's public key.
```
Comment (pfrazee) [inline README.md]: Let's s/public key/url/

File: README.md (hunk 1 above)
Comment (taravancil) [inline README.md]: How about "Start hosting your website with Dat"?

Comment (pfrazee) [inline README.md]: Yes!

**Comment by pfrazee** [comment]
💯

**Pull request merged**